dotnet/src/support/PageObjects/ByAll.cs

```diff
@@ -0,0 +1,79 @@
+﻿/*
+Copyright 2007-2015 Selenium committers
```

Comment:
copyright should actually be Software Freedom Conservancy
and probably just 2015 :)